notes/prepare-for-1.0.0-beta5.md

```diff
@@ -0,0 +1,151 @@
+﻿# Prepare for Azure Management Libraries for .NET 1.0.0-beta5#
+
+Steps to migrate code that uses Azure Management Libraries for .NET from beta 4 to beta 5 …
+
+> If this note missed any breaking changes, please open a pull request.
+
+# `Create()` defaults to Managed Disks#
+
+In `IVirtualMachine, IVirtualMachineScaleSet` and `IVirtualMachineScaleSetVM` the OS and data disks getters and withers **default** to managed disks.
+
+The withers and getters for storage account based (unmanaged) OS and data disks are **renamed** to include the term `unmanaged`.
+
+## `Create()` creates unmanaged disks on explicit requests##
+Starting in 1.0.0-beta5, if you like to continue to use the storage account based (unmanaged) operating system and data disks, you may use `WithUnmanagedDisks()` in the `Define() ... Create()` method chain. 
+
+The following sample statement creates a virtual machine with an unmanaged operating system disk:
+    
+    azure.VirtualMachines.Define("myLinuxVM")
+       .WithRegion(Region.USEast)
+       .WithNewResourceGroup(rgName)
+       .WithNewPrimaryNetwork("10.0.0.0/28")
+       .WithPrimaryPrivateIpAddressDynamic()
+       .WithNewPrimaryPublicIpAddress("mylinuxvmdns")
+       .WithPopularLinuxImage(KnownLinuxVirtualMachineImage.UbuntuServer16_04_Lts)
+       .WithRootUsername("tirekicker")
+       .WithSsh(sshKey)
+       // Unmanaged disks - uses Storage Account
+       .WithUnmanagedDisks()
+       .WithSize(VirtualMachineSizeTypes.StandardD3V2)
+       .Create();
+
+For additional sample code, please see <a href="https://github.com/Azure-Samples/compute-dotnet-manage-virtual-machine-with-unmanaged-disks">Manage Virtual Machine With Unmanaged Disks</a> ready-to-run sample. 
+
+## Converting virtual machines with storage account based disks to use managed disks
+ ##
+You can convert a virtual machine with unmanaged disks (Storage Account based) to managed disks with a single reboot.
+
+    var virtualMachines = azure.VirtualMachines.List();
+    foreach (var virtualMachine : virtualMachines) {
```

Comment:
Please use C# codding style - i.e. curly bracket on new line. Should be fixed on this and the next lines